.github/workflows/deploy.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - master
       - development
+  workflow_dispatch:
```

Comment:
Not sure how this works? Or what it does?

Reply:
I moved it to another PR
https://github.com/nimblehq/nimble-survey-web/pull/33